hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1028,6 +1029,7 @@ private static void initializeSecurityIfNeeded(OzoneConfiguration conf,
    */
   public static boolean scmInit(OzoneConfiguration conf,
       String clusterId) throws IOException {
+    ExitUtils.disableSystemExit();
```

Comment:
Why is `disableSystemExit` needed in production code?  (I guess this has to do with `stop` vs. `close` in state machine.)

Reply:
yes, it is . when initiating scm , scm ratis server will be created and then wait for the leader is elected, and then scm ratis server will be closed, which will close statemachine. since  `statemanchine#close` will terminate scm process from this patch, i think we can use `disableSystemExit` to avoid this termination, so that scm can finish this initialization completely.